src/home/actions.ts

```diff
@@ -33,7 +35,16 @@ export interface RefreshBalancesBalancesAction {
   type: Actions.REFRESH_BALANCES
 }
 
-export type ActionTypes = SetLoadingAction | UpdateNotificationsAction | DismissNotificationAction
+interface UpdateCleverTapInboxMessagesAction {
+  type: Actions.UPDATE_CLEVERTAP_INBOX_MESSAGES
```

Comment:
```suggestion
  type: Actions.CLEVERTAP_INBOX_MESSAGES_RECIEVED
```

(and also the interface name)